conda/activate.py

```diff
@@ -234,15 +308,18 @@ def build_activate(self, env_name_or_prefix):
             #  i.e. step back down
             return self.build_deactivate()
 
-        activate_scripts = self._get_activate_scripts(prefix)
+        assert 0 <= old_conda_shlvl <= max_shlvl
+
+        args = {}
```

Comment:
I'd rather not have these unrelated `args = {}` changes in this PR.
It maybe would make sense to do that as part of https://github.com/conda/conda/issues/6595.

Reply:
It was introduced in part because of the unexport idea but also because I know that the prompt support is currently limited to posix and csh. This change was done for future needs in the update prompt function. But since that's being reverted I guess this will be reverted as well.